app/lib/shared/analyzer_client.dart

```diff
@@ -65,4 +86,43 @@ class AnalysisView {
     list.sort();
     return list;
   }
+
+  /// Returns the raw health score between -1.0 and 1.0
+  /// These need to be normalized for search and frontend purposes.
+  double get health {
```

Comment:
We will migrate every non-trivial calculation from the `AnalysisView` to `pana` once we are happy with its use. However as this is entirely experimental and we don't know yet how we are going to use it, doing these in `pana` first would introduce a much bigger development roundtrip.